### ISSUE TITLE
Waterfall + Sponge(Forge 2768) doesnt work

### ISSUE THREAD

**Issue by astetyn** [PaperMC/Waterfall#360]
Hello,

I am using: `forge-1.12.2-14.23.5.2768` `spongeforge-1.12.2-2768-7.1.5` and the latest `waterfall 261` + using` ViaVersions 1.6.0 (latest)`

I have quite big server with Lobby on 1.11.2, Sponge(forge) on 1.12.2 and I was still using BungeeCord and it worked fine with SpongePls plugin. Recently I wanted to move the Lobby on 1.13.2 so I updated BungeeCord and realised, that SpongePls isnt supported and my Forge server, which is running on 1.12.2 doesnt connect with bungee. So I tried Waterfall. 

When I tried to connect to the forge server from lobby, this happend on **Sponge(forge) side**:
 `[09:40:52] [Netty Epoll Server IO #23/ERROR] [FML]: NetworkDispatcher exception`
`[Server thread/INFO] [net.minecraft.network.NetHandlerPlayServer]: AstetYne lost connection: Timed out`

and on **Waterfall side** just this: 
`[AstetYne|/***.***.100.205:64909] <-> ServerConnector [Technic] has connected`
`[AstetYne|/***.***.100.205:64909] <-> ServerConnector [Technic] has disconnected`
and nothing happend with me. I was just standing in lobby.

Using same configs as on previous bungee which was working -> BungeeCord enabled on Sponge side, ip-forwarding enabled, online-mode disabled.

**Comment by electronicboy** [comment]
Timed out means that it took too long to login (and respond to keepalives), there are mods, e.g. randompatches, and system variables designed (e.g. `-Dfml.readTimeout=` defaulting to 30) to combat some changes in 1.12.2 which means that clients can easily time out when mods take too long to allow a client to connect

**Comment by astetyn** [comment]
Thanks for reply!

I tried randompatches.
But nope. I have set it on both sides to 300 seconds. I think thats enough time for connection. After this time, the waterfall returns to client `Exception Connecting: ReadTimeoutException: null`. Problem must be somewhere else. I think some data are missing which Forge is expecting to get from Waterfall.
Also, all is running in offline mode if this could be issue.

**EDIT:** My bad, I am an idiot. I have never noted the setting `forge_support: true` in the bungee config. Sorry again and thanks for your time.
